tests/DependencyInjection/ClassNotFoundException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\Chimera\Tests\DependencyInjection;
+
+final class UnableToLoadPackage extends \RuntimeException
+{
+    const ERROR_MESSAGE = 'Application cannot be loaded because: %s, is not defined in you application.';
```

Comment:
Please add visibility here

Reply:
Maybe `Application cannot be registered because no implementation for "%s" was found.` is better